lib/cash-in/cash-in-tx.js

```diff
@@ -49,13 +50,14 @@ function post (machineTx, pi) {
             notifier.notifyIfActive('compliance', 'blacklistNotify', r.tx, true)
             addressReuse = true
           }
-          return postProcess(r, pi, blacklisted, addressReuse, failedWalletScore)
+          return postProcess(r, pi, blacklisted, addressReuse, walletScore)
         })
         .then(changes => cashInLow.update(db, updatedTx, changes))
         .then(tx => _.set('bills', machineTx.bills, tx))
         .then(tx => _.set('blacklisted', blacklisted, tx))
         .then(tx => _.set('addressReuse', addressReuse, tx))
-        .then(tx => _.set('failedWalletScore', failedWalletScore, tx))
+        .then(tx => _.set('validWalletScore', _.isNil(walletScore) ? true : walletScore.isValid, tx))
```

Comment:
if walletScore is null, we want to set validWalletScore to true?

Reply:
Hmm, I think I got it. It's null when we can't access the client, and since we can't validate the wallet, we proceed normally, is that right?